### PR TITLE
Feature/adding new links to the instructor sidebar

### PR DESCRIPTION
**Pull request by moua0061**
## Description

Added and removed links on the side bar when the instructor logs in. The feature should have overview, courses, course, messages and logout links for the side bar.
Co-authors: @MaryAngelique @Raj-04 @sagun1151 @Adamh1990 @mark-escosura 

## Loom Video

https://www.loom.com/share/02cfdbe64c564dc39362582494640c12

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

File: src/components/common/Banner.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import React from 'react';
 function bannerContent(title, content) {
   return (
     <div className="dashboard-banner">
-      <div className="banner-conent">
+      <div className="banner-content">
```
Comment (jmerz826) [inline src/components/common/Banner.js:6]: good catch: typo

File: src/components/pages/Classroom/StudentCard.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Card, Avatar, Badge } from 'antd';
+
+const { Meta } = Card;
+
+const StudentCard = ({ student }) => {
```
Comment (jmerz826) [inline src/components/pages/Classroom/StudentCard.js:6]: Is this relevant to the instructor sidebar?

File: src/components/pages/InstructorAddCourse/InstructorAddCourseForm.js [hunk 3]
```diff
@@ -9,7 +9,7 @@ import schema from './InstructorAddCourseFormSchema';
 const { Option } = Select;
 
 //dummy data
-const courses = ['Javascript', 'HTML', 'CSS'];
+const programs = ['CoderYoga', 'CoderCamp', 'CoderSitters'];
```
Comment (jmerz826) [inline src/components/pages/InstructorAddCourse/InstructorAddCourseForm.js:12]: good consistency!

File: src/components/pages/InstructorAddCourse/InstructorAddCourseForm.js [hunk 4]
```diff
@@ -39,6 +39,7 @@ const initialClassDataState = {
 
 const initialClassDataStateFormErrors = {
   course_type: '',
```
Comment (jmerz826) [inline src/components/pages/InstructorAddCourse/InstructorAddCourseForm.js:41]: program?

File: src/components/pages/InstructorAddCourse/InstructorAddCourseForm.js [hunk 5]
```diff
@@ -51,7 +52,7 @@ const initialClassDataStateFormErrors = {
   location: '',
 };
 
-const InstructorAddCourseCards = props => {
+const InstructorAddCourseForm = props => {
```
Comment (jmerz826) [inline src/components/pages/InstructorAddCourse/InstructorAddCourseForm.js:55]: good. much better name

File: src/components/pages/InstructorAddCourse/InstructorAddCourseForm.js [hunk 6]
```diff
@@ -90,9 +91,19 @@ const InstructorAddCourseCards = props => {
...
+          justifyContent: 'center',
+        }}
+      >
+        <h1
+          style={{
+            marginTop: 30,
+            fontSize: 20,
+          }}
+        >
+          You can submit a new course below!
```
Comment (jmerz826) [inline src/components/pages/InstructorAddCourse/InstructorAddCourseForm.js:106]: much better naming consistency. Instructors are adding courses, not programs

File: src/styles/InstructorStyles/index.less [hunk 7]
```diff
@@ -37,12 +37,11 @@
 //banner style
 .dashboard-banner {
   background-color: #006c72;
-
   padding: 10px 10px;
   height: 25vh;
 }
 
-.banner-conent {
+.banner-content {
```
Comment (jmerz826) [inline src/styles/InstructorStyles/index.less:44]: good catch

File: src/components/pages/InstructorHome/InstructorSidebar.js [hunk 8]
```diff
@@ -35,28 +33,20 @@ function InstructorSidebar() {
...
-          key="1"
-          icon={<ThunderboltOutlined />}
-          className="dashboard-logo"
-        >
-          Coderheroes
-        </Menu.Item>
-
         <Menu.Item key="2" icon={<HomeOutlined />}>
-          <a href="/instructor">Dashboard</a>
+          <a href="/instructor">Overview</a>
```
Comment (jmerz826) [inline src/components/pages/InstructorHome/InstructorSidebar.js:37]: good. these routes look better

File: src/components/pages/InstructorHome/ClassCard.js [hunk 9]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Card, Avatar, Badge } from 'antd';
+
+const { Meta } = Card;
+
+const ClassCard = ({ courses }) => {
```
Comment (jmerz826) [inline src/components/pages/InstructorHome/ClassCard.js:6]: again. This doesn't feel relevant to the instructor sidebar

File: src/styles/ClassroomStyles/card.less [hunk 10]
```diff
@@ -0,0 +1,20 @@
+/* Student Card */
+
+.student__card {
```
Comment (jmerz826) [inline src/styles/ClassroomStyles/card.less:3]: not related to sidebar

File: src/styles/ClassroomStyles/classroom.less [hunk 11]
```diff
@@ -0,0 +1,56 @@
+/* Classroom */
```
Comment (jmerz826) [inline src/styles/ClassroomStyles/classroom.less:1]: not related to sidebar

**Review by jmerz826 — APPROVED** [review]
Overall nice job. My one main critique would be that there are way too many changes and updates in this branch/PR that aren't related to the the instructor sidebar.  The PR video overview and the PR description exclusively discussed the styling and reordering of the sidebar, but a good portion of this code is modifying student cards and classrooms.  It all looks fine, but should be a separate pull request!

File: src/components/common/Banner.js (hunk 1 above)
Comment (ryan-hamblin) [reply src/components/common/Banner.js:6]: Yo! That's an awesome catch!

File: src/components/pages/Classroom/index.js [hunk 12]
```diff
@@ -0,0 +1,100 @@
...
+
+const students = [
+  {
+    id: 'a24efb',
+    name: 'Margery',
+    image:
+      'https://images.unsplash.com/photo-1517841905240-472988babdf9?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=687&q=80',
+  },
+  {
+    id: 'a24efb',
```
Comment (ryan-hamblin) [inline src/components/pages/Classroom/index.js:17]: Love the dummy data. It might look a little nicer presented if you make the data dynamic and versatile. The avatars on that page looked so good so getting a dozen or so different sets of data would be awesome.

File: src/components/pages/Classroom/index.js [hunk 13]
```diff
@@ -0,0 +1,100 @@
...
+                <p>students lack any feedback badges</p>
+              </Row>
+            </Card>
+            <Button className="classroom_feedback__button">
+              GIVE FEEDBACK
+            </Button>
+          </Row>
+        </Content>
+      </Layout>
+    </>
```
Comment (ryan-hamblin) [inline src/components/pages/Classroom/index.js:96]: This `Fragment` seems to be irrelevant if `<Layout/>` is the wrapper component. If you need a sibling to `Layout` then a `Fragment` makes sense.

**Review by ryan-hamblin — DISMISSED** [review]
Great stuff so far! I think a few changes, fixing the typos your peers have mentioned and the `</>` fixes would go a long way :)

**Review by Raj-04 — APPROVED** [review]
Overall! Great work

**Review by MaryAngelique — APPROVED** [review]
